pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1839,7 +1841,7 @@ func (ctrl *PersistentVolumeController) provisionClaimOperationExternal(
 		return provisionerName, err
 	}
 	claim = newClaim
-	msg := fmt.Sprintf("waiting for a volume to be created, either by external provisioner %q or manually created by system administrator", provisionerName)
+	msg := fmt.Sprintf("Requested external provisioner %q for a volume to be created, or waiting manually created volume by system administrator", provisionerName)
```

Comment:
Nit: To be grammatical, I think the second half of the message should be:
> "or waiting for a volume to be manually created by system administrator"

But really, the whole message would be easier to read if it said:
>"Waiting for a volume to be created (either by the external provisioner %q, or manually by the system administrator)"